backend/apps/notifications/migrations/0005_notification_delivery.py

```
@@ -0,0 +1,68 @@
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("notifications", "0004_rename_notificatio_recipie_4e3567_idx_idx_recipientis_read"),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)